src/main/resources/dbmigration/mysql/V3__createTables.sql

```diff
@@ -0,0 +1,81 @@
+drop table user;
+
+CREATE TABLE `user` (
+  `userId` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `firstName` varchar(255) CHARACTER SET utf8 COLLATE utf8_bin DEFAULT NULL,
+  `lastName` varchar(255) CHARACTER SET utf8 COLLATE utf8_bin DEFAULT NULL,
+  `note` varchar(255) CHARACTER SET utf8 COLLATE utf8_bin DEFAULT NULL,
+  `virtualBalance` decimal(14,2) NOT NULL DEFAULT '0.00',
+  `winnings` decimal(14,2) NOT NULL DEFAULT '0.00',
```

Comment:
i don't see a need to store this information. this can be queried from the Transactions table